backend/index.js

```diff
@@ -17,6 +21,7 @@ const runCrawler = async (crawler) => {
   });
   child.on('exit', (exitCode, signal) => {
     logger.warn(`Crawler ${crawler} exit with code: ${exitCode} and signal: ${signal}`);
+    status.set(name, `exit code ${exitCode}, signal ${signal}`);
```

Comment:
You should update the status in case of exit for any reason and not only `exit`

Reply:
Port 3000 is used by the  client-side application

Reply:
When a javascript error occurs the `exit` event is triggered.
`SIGUSER1`, `SIGUSER2` are used in Nodjes by user-defined conditions in the code.  Node.js chose `SIGUSR1` to use this event to start the built-in debugger.
Only `uncaughtException` makes sense to add for some specific edge cases.

Reply:
Discussed and leaving as is.